Configuration.props

```diff
@@ -16,6 +16,8 @@
     <!-- Used by the `build-tools/create-vsix` build so that `Mono.Android.Export.dll`/etc. are only included *once* -->
     <!-- Should correspond to the first value from `$(API_LEVELS)` in `build-tools/scripts/BuildEverything.mk` -->
     <AndroidFirstFrameworkVersion Condition="'$(AndroidFirstFrameworkVersion)' == ''">v4.4</AndroidFirstFrameworkVersion>
+    <AndroidFirstApiLevel Condition="'$(AndroidFirstApiLevel)' == ''">19</AndroidFirstApiLevel>
+    <AndroidFirstPlatformId Condition="'$(AndroidFirstPlatformId)' == ''">$(AndroidFirstApiLevel)</AndroidFirstPlatformId>
```

Comment:
So we wouldn't need these if we used the latest `android.jar`: https://github.com/xamarin/xamarin-android/pull/2536#discussion_r262931625

Reply:
see https://github.com/xamarin/xamarin-android/pull/2536#discussion_r263488223